scripts/tests/test_singularity_cmd.bats

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bats
+#
+# This test uses the Bash Automated Testing System
+# See: https://github.com/bats-core/bats-core
+#
+# Install instructions:
+#   git clone https://github.com/bats-core/bats-core.git
+#   cd bats-core
+#   sudo ./install.sh /usr/local
+#
+# Several ways to run the test:
+#   ./test_singularity_cmd.bats
+#   bats test_singularity_cmd.bats
+#   bats .
+
+@test "verifying arguments passed to singularity_cmd Docker shim" {
+    export REPRONIM_USE_DOCKER=1
+    run ../singularity_cmd \
+        exec arg-test.simg /singularity "foo bar" blah 45.5 /dir
```

Comment:
would you be kind to add one more (or just include to existing one) obscure one with `&` `;` and `'${something}'` in it/them? ;-)